apps/design-system/src/pages/view-preview/view-settings.tsx

```diff
@@ -43,8 +43,9 @@ const ViewSettings: FC<ViewSettingsProps> = ({ routes }) => {
   return (
     <aside className={clsx(css.viewSettings, 'shadow-4')} data-show={showSettings ? 'show' : 'hide'}>
       <Button
-        variant="link_accent"
-        size="icon"
+        variant="link"
+        size="sm"
+        iconOnly
```

Comment:
Radix uses a separate component for `IconButton`, which keeps the API very clean, and allows for combinations of variants and icons more easily.
Thoughts on adopting a similar pattern?
https://www.radix-ui.com/themes/docs/components/icon-button

Reply:
cc: @cxramina